### PR TITLE
chore: run.js supports new structure

### PR DESCRIPTION
**Pull request by swatDong**
Make run-utils to support new folder structure.
Follow the same logic as #7321 to load `projectId` as key.

### CODE REVIEW

File: packages/run-utils/src/index.ts [hunk 1]
```diff
@@ -44,13 +50,25 @@ export async function loadEnv(
...
+  let projectId = "";
+  if (await fs.pathExists(settingsPath)) {
+    projectId = await readSettings(settingsPath);
+    if (!projectId) {
+      throw new Error(`trackingId is missing in ${settingsFileName}`);
+    }
+  } else if (await fs.pathExists(yamlPath)) {
+    projectId = await readYaml(yamlPath);
+    if (!projectId) {
+      throw new Error(`projectId is missing in ${projectYamlName}`);
```
Comment (kuojianlu) [inline packages/run-utils/src/index.ts:64]: trackingId

File: packages/run-utils/src/index.ts [hunk 2]
```diff
@@ -5,23 +5,29 @@ import Cryptr from "cryptr";
...
+async function readSettings(filePath: string): Promise<string> {
+  if (!(await fs.pathExists(filePath))) {
+    throw new Error(`${filePath} does not exist.`);
+  }
+
+  const settings = await fs.readJson(filePath);
+  return settings.trackingId;
+}
+
+async function readYaml(filePath: string): Promise<string> {
```
Comment (kuojianlu) [inline packages/run-utils/src/index.ts:23]: How about `readProjectIdFromYaml`?

File: packages/run-utils/src/index.ts [hunk 3]
```diff
@@ -5,23 +5,29 @@ import Cryptr from "cryptr";
...
 
 const settingsFolderName = "teamsfx";
 const settingsFileName = "settings.json";
+const projectYamlName = "teamsapp.yml";
 
-async function readSettings(projectPath: string): Promise<Settings> {
-  const settingsPath = path.join(projectPath, settingsFolderName, settingsFileName);
-  if (!(await fs.pathExists(settingsPath))) {
-    throw new Error(`${settingsPath} does not exist.`);
+async function readSettings(filePath: string): Promise<string> {
```
Comment (kuojianlu) [inline packages/run-utils/src/index.ts:14]: How about `readTrackingIdFromSettings`?

Comment (swatDong) [reply packages/run-utils/src/index.ts]: Updated

File: packages/run-utils/src/index.ts (hunk 2 above)
Comment (swatDong) [reply packages/run-utils/src/index.ts]: Updated

**Review by kuojianlu — APPROVED** [review]

**Pull request merged**